funnel/extapi/boxoffice.py

```diff
@@ -31,18 +36,24 @@ def get_tickets(self, ic):
             for line_item in order.get('line_items'):
                 if line_item.get('assignee'):
                     status = unicode(line_item.get('line_item_status'))
-                    tickets.append({
-                        'fullname': line_item.get('assignee').get('fullname', ''),
-                        'email': line_item.get('assignee').get('email'),
-                        'phone': line_item.get('assignee').get('phone', ''),
-                        'twitter': extract_twitter_handle(line_item.get('assignee').get('twitter', '')),
-                        'company': line_item.get('assignee').get('company'),
-                        'city': line_item.get('assignee').get('city', ''),
-                        'job_title': line_item.get('assignee').get('jobtitle', ''),
-                        'ticket_no': unicode(line_item.get('line_item_seq')),
-                        'ticket_type': line_item.get('item', {}).get('title', '')[:80],
-                        'order_no': unicode(order.get('invoice_no')),
-                        'status': status
-                        })
+                    tickets.append(
+                        {
+                            'fullname': line_item.get('assignee').get('fullname', ''),
+                            'email': line_item.get('assignee').get('email'),
+                            'phone': line_item.get('assignee').get('phone', ''),
+                            'twitter': extract_twitter_handle(
+                                line_item.get('assignee').get('twitter', '')
+                            ),
+                            'company': line_item.get('assignee').get('company'),
+                            'city': line_item.get('assignee').get('city', ''),
+                            'job_title': line_item.get('assignee').get('jobtitle', ''),
+                            'ticket_no': unicode(line_item.get('line_item_seq')),
+                            'ticket_type': line_item.get('item', {}).get('title', '')[
+                                :80
+                            ],
```

Comment:
This one is a bit ridiculous. Maybe ignore it.

Reply:
That's because it hit the 88 column limit.